components-core/src/main/java/com/adyen/checkout/components/core/internal/ui/model/SessionParams.kt

```diff
@@ -12,10 +12,16 @@ import androidx.annotation.RestrictTo
 import com.adyen.checkout.components.core.Amount
 import java.util.Locale
 
+/**
+ * Object that holds values set during sessions setup call.
+ * [SessionParams] values should always have higher priority than values set in client side configurations. Otherwise
```

Comment:
This is correct for most attributes but there are exceptions. For example `shopperLocale` can be overridden by a client side configuration

Reply:
Adjusted it a bit to give more context